src/components/Sidebar/components/NavItem/NavItem.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme }) => css`
+  label: nav-item;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  height: auto;
+  margin: ${theme.spacings.mega};
+  padding: ${theme.spacings.bit};
+  cursor: pointer;
+  color: ${theme.colors.n500};
+  fill: ${theme.colors.n500};
+`;
+
+const hoverStyles = ({ theme, selected }) => css`
+  label: nav-item__hover;
+  @media (hover: hover) {
+    &:hover {
+      color: ${!selected && theme.colors.n300};
+      fill: ${!selected && theme.colors.n300};
+    }
+  }
+`;
+
+const activeStyles = ({ theme, selected }) => css`
+  label: nav-item__active;
+  color: ${selected && theme.colors.n100};
+  font-weight: ${selected && theme.fontWeight.bold};
+  &:active {
```

Comment:
Mhm, I'm not sure if the `:active` styles belong here instead of the base styles. The `:active` state refers to the item being focused with the enter key or the mouse pointer pressed down. It's unrelated to an item's selected state.